tensorboard/plugins/hparams/list_session_groups.py

```diff
@@ -565,7 +565,7 @@ class _MetricIdentifier:
     Attributes:
       group: Metric group corresponding to the dataset on which the model was
         evaluated.
-      tag: String tag assoicated with the metric.
+      tag: String tag associated with the metric.
```

Comment:
For a revert, I'd avoid introducing any unrelated changes, even if they're just a typo fix. Part of the idea w/ a revert is that it shouldn't really require review other than seeing that it's a rollback of a previous change, so it's more important than usual to avoid introducing any unrelated deltas.

Reply:
Got it, reverted the typo fix.